jenkins/jenkins2-ha-agents.yaml

```diff
@@ -530,6 +534,19 @@ Resources:
       LoadBalancerArn: !Ref MasterELB
       Port: 80
       Protocol: HTTP
+  MasterELBListenerHTTPS:
```

Comment:
If we've altered the security group to only allow ingress on port 443 and not on port 80 any longer, how come we still need the ELB listener on port 80 above?